roxctl/declarativeconfig/configmap/configmap.go

```diff
@@ -0,0 +1,99 @@
+package configmap
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/errox"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Flag names exposing the config map name and the namespace where the configmap resides.
+const (
+	ConfigMapFlag = "config-map"
+	NamespaceFlag = "namespace"
+)
+
+// WriteToConfigMap writes bytes containing a YAML string to a config map under a specific key.
+// The Kubernetes context will be inferred from either the $KUBECONFIG variable or the $HOME/.kube/config path.
+func WriteToConfigMap(configMap, namespace, key string, yaml []byte) error {
+	client, namespaceFromConfig, err := createK8SClient(namespace)
+	if err != nil {
+		return err
+	}
+
+	cm, err := client.CoreV1().ConfigMaps(namespaceFromConfig).Get(context.TODO(), configMap, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "retrieving config map %s", configMap)
+	}
+
+	if cm.Data == nil {
+		cm.Data = map[string]string{}
+	}
+	cm.Data[key] = string(yaml)
+
+	if _, err := client.CoreV1().ConfigMaps(namespaceFromConfig).Update(context.TODO(), cm, metav1.UpdateOptions{}); err != nil {
+		return errors.Wrapf(err, "updating config map %s", configMap)
+	}
+	return nil
+}
+
+// ReadFromConfigMap will read all keys from a config map and return them in byte format.
+// The Kubernetes context will be inferred from either the $KUBECONFIG variable or the $HOME/.kube/config path.
+// Note: the binary data within the config map will be skipped, as the output shall be UTF-8 compatible.
+func ReadFromConfigMap(configMap, namespace string) ([][]byte, error) {
+	client, namespaceFromConfig, err := createK8SClient(namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	cm, err := client.CoreV1().ConfigMaps(namespaceFromConfig).Get(context.TODO(), configMap, metav1.GetOptions{})
+	if err != nil {
+		return nil, errors.Wrapf(err, "retrieving config map %s", configMap)
+	}
+	var contents [][]byte
```

Comment:
super nit: prealloc? 

Maybe we should enable linter for that although I think in CLI it makes little sense to optimize this